Jenkinsfile

```diff
@@ -1,131 +1,157 @@
 pipeline {
-  agent {
-    dockerfile {
-      additionalBuildArgs '--build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g)'
-      args '-m 60g'
-    }
-  }
+  agent none
   options {
     ansiColor('xterm')
+    skipDefaultCheckout()
   }
   stages {
-    stage("Init title") {
+    stage('Kill old builds') {
+      agent none
       when { changeRequest() }
       steps {
-        script {
-          currentBuild.displayName = "PR ${env.CHANGE_ID}: ${env.CHANGE_TITLE}"
-        }
-      }
-    }
-    stage('Dependencies') {
-      steps {
-        sh '''
-          export PATH=$HOME/.local/bin:$HOME/.cargo/bin:$PATH
-          make all-deps -B
-          make split-tests -B
-        '''
-      }
-    }
-    stage('Build') {
-      steps {
-        sh '''
-          export PATH=$HOME/.local/bin:$HOME/.cargo/bin:$PATH
-          make build build-llvm build-haskell build-node -j4 -B
-        '''
+        build job: 'rv-devops/master', parameters: [ string(name: 'BUILD_JOB_NAME', value: "${env.JOB_NAME}")
+                                                   , string(name: 'BUILD_JOB_NUMBER', value: "${env.BUILD_NUMBER}")
+                                                   ]
       }
     }
-    stage('Test Execution') {
-      failFast true
-      parallel {
-        stage('Conformance (OCaml)') {
+    stage('Run CI') {
+      agent { label 'docker' }
+      stages {
+        stage('Init title') {
+          when { changeRequest() }
```

Comment:
let's put this up in the agent none segment. we don't need a whole extra stage for this separate from the build killer stage.